examples/animation-controls/animation-controls.js

```diff
@@ -0,0 +1,63 @@
+
+animationNames = {
+  attack: 'Armature|TRex_Attack',
+  death: 'Armature|TRex_Death',
+  idle: 'Armature|TRex_Idle',
+  jump: 'Armature|TRex_Jump',
+  run: 'Armature|TRex_Run',
+  walk: 'Armature|TRex_Walk',
+};
+
+
+
+updateAnimationMixer = () => {
+
```

Comment:
Nit pick :) What coding style is this? You put a blank line at the top of the function but not the bottom? But you don't do it every time, so the style is not consistent. I'll personally don't put a blank line at the beginning of a function.
I know the threejs coding style that want a blank line at the beginning and at the end, but you don't use this style here.